Source/CombatExtended/CombatExtended/Projectiles/BulletCE.cs

```diff
@@ -47,10 +49,12 @@ protected override void Impact(Thing hitThing)
                 // launcher being the pawn equipping the weapon, not the weapon itself
                 int damageAmountBase = def.projectile.GetDamageAmount(CE_Utility.GetWeaponFromLauncher(launcher));
```

Comment:
Vanilla method, does this return correct values for projectiles?

Reply:
I don't know, I only have a basic understanding of Rimworlds methods and classes.
Will try to check, I assume not, since they are defined in a custom config.

Reply:
Hmm seems like this was there before my changes, I could use projectilePropsCE instead.

Reply:
Seems like it increases the damage based on a multiplier.
Source:
https://github.com/josh-m/RW-Decompile/blob/master/Verse/ProjectileProperties.cs#L76

Reply:
Yeah, double-checked it and this is the correct method to use.